package.json

```diff
@@ -30,28 +30,26 @@
         }
     },
     "dependencies": {
-        "@11ty/eleventy": "1.0.0",
+        "@11ty/eleventy": "1.0.1",
         "@11ty/eleventy-navigation": "0.3.2",
-        "@11ty/eleventy-plugin-syntaxhighlight": "3.2.2",
+        "@11ty/eleventy-plugin-syntaxhighlight": "4.0.0",
         "eleventy-plugin-fluid": "1.0.0",
-        "infusion": "4.0.0",
```

Comment:
Are you sourcing Infusion from `eleventy-plugin-fluid`?

Reply:
Infusion isn't directly referenced by anything in the website itself any more (now that the call to `fluid.allocateSimpleId()` was removed from #85), though I was able to find the following versions being pulled in as subdeps in the `package-lock`:

| package | version |
|----------|---------|
| eleventy-plugin-fluid | `4.x` |
| fluid-glob | `3.0.0-dev.20210312T233557Z.0b016a6dc.FLUID-6580` |
| fluid-lint-all | `4.0.0` |
| markdownlint-config-fluid | `3.0.0-dev.20210312T233557Z.0b016a6dc.FLUID-6580` |

So the short answer seems to be yes, but it also seems to be sourced from several other packages as well.